src/Compilers/Core/CodeAnalysisTest/Text/TextChangeTests.cs

```diff
@@ -636,6 +636,45 @@ public void TestMergeChanges_Overlapping_NewInsideOld()
             Assert.Equal("Cool ", changes[0].NewText);
         }
 
+        [Fact]
```

Comment:
>Fact [](start = 9, length = 4)

Please add a `WorkItem` attribute to each of these three tests and the last test. 
```
[WorkItem(22289, "[https://github.com/dotnet/roslyn/issues/22289](https://github.com/dotnet/roslyn/issues/22289)")]
```

Reply:
Looks like the link might have been auto-formatted, did you mean something like: 

```cs
[WorkItem(22289, "https://github.com/dotnet/roslyn/issues/22289")]
```

Reply:
Done.